include/core/federated/federate.h

```diff
@@ -554,4 +560,37 @@ void lf_synchronize_with_other_federates();
  */
 bool lf_update_max_level(tag_t tag, bool is_provisional);
 
+/**
+ * @brief Stop the execution of a federate.
+ * Every enclave within the federate will stop at one microstep later than its
+ * current tag. Unlike lf_request_stop(), this process does not require any 
+ * involvement from the RTI, nor does it necessitate any consensus.
+ * 
+ * This function is particularly useful for testing transient federates.
+ */
+void lf_stop();
+    
+/**
+ * @brief Return the directory containing the executables of the individual
+ * federates.
+ */
+char* lf_get_federates_bin_directory();
```

Comment:
Exposing information about the file system location at compile time to the program seems like a pretty bad idea to me. Paths can change, binaries get deployed to other systems. If it really is needed, it should be provided as a parameter at runtime.